### PR TITLE
Added a module to set up git

### PR DESCRIPTION
**Pull request by VickySteeves**
Hi! I added a short lesson on setting up git locally (going through `config` and briefly explaining `global`) so learners can do it with us (previously, they did it on their own).

### CODE REVIEW

File: _episodes/02-setup.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- "Configure `git` for the first time on your machine"
+keypoints:
+- "You cannot use `git` without telling it your name and email address"
+---
+
+### Using Git
+
+One of the main barriers to getting started with git is the language. Although some of the language used in git is 
+fairly self-explanatory, other terms are not so clear. The best way to get to learn the language - which consists of a 
+number of verbs such as `add`, `commit` and `push` (precdeded by the word 'git') - is by using it, which is what we will be doing during this
```
Comment (remram44) [inline _episodes/02-setup.md]: typo: "precdeded"

File: _episodes/02-setup.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+### Git Help and Manual
+
+Always remember that if you forget a `git` command, you can access the list of commands by using `-h` and access the Git manual by using `--help` :>
+
+~~~
+$ git config -h
+$ git config --help
+~~~
+{: .bash}
+{: .callout}
```
Comment (remram44) [inline _episodes/02-setup.md]: I think you have to blockquote the whole thing to use the `.callout` class, [like is done here](https://github.com/swcarpentry/sql-novice-survey/blame/7f0fe00eb4409dfb2b0e5ef2d54177b0cc4f1654/_episodes/01-select.md#L35-L44). You also need the titles to be level 2 (`##` not `###`) for some reason.

**Comment by katrinleinweber** [comment]
Dear @VickySteeves, Because of a transition of the maintainer team here, this PR has not received the attention it deserved, I'm sorry :-( Please have a look at #42 for further discussion about this whole setup topic.

**Comment by dcmcand** [comment]
This section has already been added, so I am going to close this PR.

**Pull request closed without merging**